ci-operator/step-registry/hypershift/kubevirt/baremetalds/conformance-calico/hypershift-kubevirt-baremetalds-conformance-calico-workflow.metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"path": "hypershift/kubevirt/baremetalds/conformance-calico/hypershift-kubevirt-baremetalds-conformance-calico-workflow.yaml",
+	"owners": {
+		"approvers": [
+			"davidvossel",
+			"nirarg",
+			"nunnatsa",
+			"qinqon",
+			"orenc1",
+			"LiangquanLi930"
+		],
+		"reviewers": [
+			"davidvossel",
```

Comment:
David is not in that team anymore, please update

Reply:
OK.